packages/wagmi/src/connectors/EIP6963Connector.ts

```diff
@@ -1,5 +1,6 @@
-import type { Chain, WindowProvider } from '@wagmi/core'
+import { type Chain, type WindowProvider } from '@wagmi/core'
```

Comment:
Seems like `import type { ... }` is our convention across the repo. Any specific reason to use this way?

Reply:
Prettier or something idk, I'll fix it, makes no sense in this format lol